dev/run-tests.py

```diff
@@ -687,6 +689,10 @@ def main():
         test_modules = determine_modules_to_test(changed_modules)
         excluded_tags = determine_tags_to_exclude(changed_modules)
 
+    # With this script, disable docker integration by default.
```

Comment:
Could we leverage environ in modules? https://github.com/apache/spark/blob/620af5bcdc1fbc446f5cbb539ce79e6163a8d425/dev/sparktestsupport/modules.py#L306-L308

Reply:
I didn't intentionally declare it.
If we declare it, `ENABLE_DOCKER_INTEGRATION_TESTS` is always `1`, meaning docker integration tests always run with `run-tests.py`.
Docker integration tests need Docker installed but we can't assume all the environment satisfies that.
I'm afraid that If anyone who runs `run-tests.py` on their testing environment, they fail to do after this change.

Reply:
oh @sarutak, if `ENABLE_DOCKER_INTEGRATION_TESTS` is set to `1`, it will be only set when the module is explicitly selected. That's what `ENABLE_KINESIS_TESTS` does (because Kinesis test cases requires AWS resources which we shouldn't run too often) also see: https://github.com/apache/spark/commit/76f2e393a5fad0db8b56c4b8dad5ef686bf140a4

Reply:
Oh, okay, maybe `run-tests.py --modules` might have a different logic from that. I will double check

Reply:
Ah, I got it. Thanks for the explanation.

Reply:
If `run-tests.py` runs on Jenkins, `change_modules` seems to be determined based on the changed files.
https://github.com/apache/spark/blob/5cc17ba0c7d81278a622fb474cf18f8e8530335f/dev/run-tests.py#L683-L688

So, we need to install Oracle Docker container image on the Jenkins environment or just set `ENABLE_DOCKER_INTEGRATION_TESTS` to `0` for the environment right?

Reply:
Ohh yeah .. I see we should probably run this only when we're in GitHub Action.